src/HW_2_OOP/GryffindorStudents.java

```diff
@@ -0,0 +1,32 @@
+package HW_2_OOP;
+
+public class GryffindorStudents extends HogwartsStudentGeneral {
```

Comment:
```suggestion
public class GryffindorStudent extends HogwartsStudentGeneral {
```
классы нужно называть в единственном числе